src/assets/js/fullpostpage.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Let's use `full-post-page.js` for the file name here for consistency with other files.

Reply:
okay. done.